docs/application-development/foundations.md

```diff
@@ -282,9 +271,7 @@ The Temporal TypeScript SDK API reference is published on [typescript.temporal.i
 
 #### Code samples
 
-You can find a complete list of executable code samples in [Temporal's GitHub repository](https://github.com/temporalio).
-
-The [Temporal Simple Polyglot](https://github.com/temporalio/temporal-polyglot) repository showcases how Workflow Executions, written in different languages, can send messages to each other. Go, Java, PHP, and TypeScript SDKs are represented in this sample. It also shows how to properly propagate errors, including how to do so across Workflows written in different languages. For more information, see the [Polyglot Microservice Orchestration](https://www.youtube.com/playlist?list=PLl9kRkvFJrlTLo5URV5IK6lCmiM3ir3f5) video on YouTube.
+You can find a complete list of executable code samples in [Temporal's GitHub repository](https://github.com/temporalio?q=samples-&type=all&language=&sort=).
```

Comment:
don't think this link is helpful, would remove and change below line to:

In addition to the samples listed in the language tabs below, several of the [Tutorials](https://learn.temporal.io) are backed by working applications.

or remove that too and list tutorials in the lang tabs